Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 BUNDLE_VERSION = 4.1.0
-CRC_VERSION = 0.87.0-alpha-$(BUNDLE_VERSION)
+CRC_VERSION = 0.87.0-alpha
```

Comment:
CRC releases will somehow always be tied to a version, as we need to make sure a bundle can ebe brought up with the associated CRC version. Else we would end up having to test a multitude of version combinations. What is the idea here?

Reply:
@gbraad idea is every release support the released version of bundle only (not older or newer). we still are struggling with the 30 days cert rotation which is not yet resolved for us. Most of the user are trying a older bundle version with latest version of crc and getting issue around the cert expire.